sym-lib-table

```diff
@@ -1,4 +1,5 @@
 (sym_lib_table
   (lib (name Effect-Box-rescue)(type Legacy)(uri ${KIPRJMOD}/Effect-Box-rescue.lib)(options "")(descr ""))
   (lib (name sit-oscillator)(type Legacy)(uri ${KIPRJMOD}/library/sit-oscillator.lib)(options "")(descr ""))
+  (lib (name LCD)(type Legacy)(uri "/Users/martinfarstad/Downloads/ul_FH12A10S05SH55 (3)/KiCAD/2019-09-20_12-13-46/2019-09-20_12-13-46.lib")(options "")(descr ""))
```

Comment:
I dont think this will work @mrfarstad , since it refers to your home directory. `ul_FH12A10S05SH55` should be embedded into the project instead.